File: api/Remindr.Model/Database/Reminder.cs
using System;

namespace Remindr.Model.Database
{
    public class Reminder
    {        
        public int _id { get; set; }
        public string _mobileNumber { get; set; }
        public string _message { get; set; }
        public DateTime _nextScheduledReminder { get; set; }
        public string _schedule { get; set; }
        public string[] _customSchedule { get; set; }
        public DateTime _endReminderDate { get; set; }
        public string _kind { get; set; }
        
        public Reminder(string mobileNumber, string message, DateTime nextScheduledReminder, string schedule,
                             string[] customSchedule, DateTime endReminderDate, string kind)
        {            
            _mobileNumber = mobileNumber;
            _message = message;
            _nextScheduledReminder = nextScheduledReminder;
            _schedule = schedule;
            _customSchedule = customSchedule;
            _endReminderDate = endReminderDate;
            _kind = kind;
        }

        public Reminder()
        {            
        }
    }
}

File: frontend/app/model/Reminder.scala
package model

import java.util.Date

case class Reminder(id: Int, user: String, number: String, kind: String, date: Date)
case class Appointment(reminderDate: Date,message: String, mobile: String, oneWeekNotification: Boolean, oneDayNotification: Boolean)
case class Medication (mobile: String, start: Date, end: Date, schedule: String, message: String)
case class Prescription(mobile: String, reminderdate: Date, message: String)

object Schedule {
  val Daily = "Daily"
  val Weekly = "Weekly"
}

object Kind {
  val Appointment = "Appointment"
  val Prescription = "Prescription"
  val Medication = "Medication"
}
